numpy/lib/utils.py

```diff
@@ -788,13 +788,11 @@ def lookfor(what, module=None, import_modules=True, regenerate=False,
         if kind in ('module', 'object'):
             # don't show modules or objects
             continue
-        ok = True
         doc = docstring.lower()
         for w in whats:
             if w not in doc:
-                ok = False
                 break
-        if ok:
+        else:
             found.append(name)
```

Comment:
If we're going to touch this rarely used code, I'd be tempted to change this to:
```
if all(w in doc for w in whats):
    found.append(name)
```
which is even clearer than the `for` / `else` construction

Reply:
Great, let's do that instead. I just force-pushed a new commit.